### PR TITLE
[Respects] Add ability to check previous X messages for respects

### PR DESCRIPTION
**Pull request by Injabie3**
### Type
- [x] Enhancement

### Description of the changes
As suggested by @IAmTheRedSpy, also add the ability to group respects together within a certain number of messages.

With this PR, for a new respect message, one of the following must be satisfied:
- A respect must not have existed in the current channel (determined from the moment that the bot starts up), or
- More than 30 seconds **and** more than 20 messages have passed **(NEW)**.

Currently, 20 and 30 are constants, which can be made configurable in a later PR.

### CODE REVIEW

File: cogs/respects.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
           ":purple_heart:", ":blue_heart:"]
 DEFAULT_CH_DICT = {KEY_MSG : None, KEY_TIME : None, KEY_USERS : []}
 TIME_BETWEEN = timedelta(seconds=30) # Time between paid respects.
+MESSAGES = 20 # The number of messages in between
```
Comment (KamiCreed) [inline cogs/respects.py:18]: 20 seems a bit much, unless this was mostly for testing purposes.

**Review by KamiCreed — APPROVED** [review]
Seems good though. In the far future, should probably consider making constants into easy to change config files.

File: cogs/respects.py (hunk 1 above)
Comment (Injabie3) [reply cogs/respects.py:18]: 20 messages was roughly the number of single-line messages for the last respect to scroll out of view on my screen.  Definitely will put this in a config file in the next PR, as you suggested.

Comment (KamiCreed) [reply cogs/respects.py:18]: Yea. Well, I feel like 20 messages would be a good amount of messages for a different "Press F to respect" to occur...

**Pull request merged**